tests/src/gcc/LaunchKernel.c

```diff
@@ -0,0 +1,180 @@
+
+/* Copyright (c) 2019-Present Advanced Micro Devices, Inc. All rights reserved.
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANNTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INNCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANNY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER INN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR INN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+
+/* HIT_START
+ * BUILD_CMD: gpu.o %hc -I%hip-path/include -g -c %S/gpu.cpp -o %T/gpu.o EXCLUDE_HIP_PLATFORM nvcc
+ * BUILD_CMD: launchkernel.o %cc -D__HIP_PLATFORM_HCC__ -g -I%hip-path/include -c %S/LaunchKernel.c -o %T/launchkernel.o EXCLUDE_HIP_PLATFORM nvcc
+ * BUILD_CMD: LaunchKernel %hc %T/launchkernel.o %T/gpu.o -g -Wl,--rpath=%hip-path/lib %hip-path/lib/libhip_hcc.so -o %T/%t DEPENDS gpu.o launchkernel.o EXCLUDE_HIP_PLATFORM nvcc
+ * TEST: %t EXCLUDE_HIP_PLATFORM nvcc
+ * HIT_END
+ */
+
+
+#include "hip/hip_runtime.h"
+#include "hip/hip_runtime_api.h"
+#include <stdio.h>
+#include "LaunchKernel.h"
+
+void LaunchKernelArg()
+{
+  hipError_t status = hipSuccess;
+  dim3 blocks 	    = {1,1,1};
+  dim3 threads      = {1,1,1};
+
+  printf(" -----: Launch Zero Argument Kernel :------ \n");
+  status = hipLaunchKernel(kernel, blocks, threads,NULL, 0, 0);
+  printf("hipLaunchKernel status %s\n",hipGetErrorString(status));
+
+  printf("Test Passed!!\n\n");
+}
+
+void LaunchKernelArg1()
+{
+  hipError_t status = hipSuccess;
+  int A = 0;
+  int *A_d = NULL;
+  dim3 blocks       = {1,1,1};
+  dim3 threads      = {1,1,1};
+
+  printf(" -----: Launch One Argument Kernel :------ \n");
+
+  // Allocate Device memory
+  status = hipMalloc((void**)&A_d, sizeof(int));
+  printf("hipMalloc status %s\n", hipGetErrorString(status));
```

Comment:
No need to print status of every HIP call. you can use HIPCHECK as used in other dtests.